### ISSUE TITLE
 Notification platform is unavailable error

### ISSUE THREAD

**Issue by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1772]
### Checks

- [X] I have checked that this issue has not already been reported.

### Problem Description

Please open new issue in: https://github.com/Flow-Launcher/Flow.Launcher/issues/new
1. upload log file: D:\Downloads\Flow-Launcher-Portable\FlowLauncher\app-1.11.0\UserData\Logs\1.11.0\2023-01-10.txt
2. copy below exception message

Flow Launcher version: 1.11.0
OS Version: Microsoft Windows NT 10.0.22621.0
IntPtr Length: 8
x64: True

Python Path: 
Node Path: 
Date: 01/10/2023 01:21:05
Exception:
System.Runtime.InteropServices.COMException (0x803E0105): The notification platform is unavailable. (0x803E0105)
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|20_0(Int32 hr)
   at ABI.Windows.UI.Notifications.IToastNotificationManagerStaticsMethods.CreateToastNotifier(IObjectReference _obj, String applicationId)
   at Windows.UI.Notifications.ToastNotificationManager.CreateToastNotifier(String applicationId)
   at Microsoft.Toolkit.Uwp.Notifications.ToastNotificationManagerCompat.CreateToastNotifier()
   at Microsoft.Toolkit.Uwp.Notifications.ToastContentBuilder.Show(CustomizeToast customize)
   at Microsoft.Toolkit.Uwp.Notifications.ToastContentBuilder.Show()
   at Flow.Launcher.Notification.Show(String title, String subTitle, String iconPath) in C:\projects\flow-launcher\Flow.Launcher\Notification.cs:line 35
   at Flow.Launcher.PublicAPIInstance.<>c__DisplayClass13_0.<ShowMsg>b__0() in C:\projects\flow-launcher\Flow.Launcher\PublicAPIInstance.cs:line 97
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback)
   at Flow.Launcher.PublicAPIInstance.ShowMsg(String title, String subTitle, String iconPath, Boolean useMainWindowAsOwner) in C:\projects\flow-launcher\Flow.Launcher\PublicAPIInstance.cs:line 95
   at Flow.Launcher.PublicAPIInstance.ShowMsg(String title, String subTitle, String iconPath) in C:\projects\flow-launcher\Flow.Launcher\PublicAPIInstance.cs:line 91
   at Flow.Launcher.Plugin.Explorer.ContextMenu.<>c__DisplayClass13_0.<LoadContextMenus>b__4(ActionContext _) in C:\projects\flow-launcher\Plugins\Flow.Launcher.Plugin.Explorer\ContextMenu.cs:line 134
   at Flow.Launcher.Plugin.Result.ExecuteAsync(ActionContext context) in /_/Flow.Launcher.Plugin/Result.cs:line 219
   at Flow.Launcher.ViewModel.MainViewModel.OpenResultAsync(String index) in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 276
   at CommunityToolkit.Mvvm.Input.AsyncRelayCommand.AwaitAndThrowIfFailed(Task executionTask)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

### To Reproduce

1. ...
2. ...
3. ...

### Flow Launcher Version

v1.11.0

### Windows Build Number

10.0.22621 Build 22621

### Error Log

<details>

```shell

Replace this line with the important log contents.

```
</details>

**Comment by raindropsfromsky** [comment]
Here is the log file:
[2023-01-10.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/10378908/2023-01-10.txt)

**Comment by raindropsfromsky** [comment]
When the app crashed, I was just trying to change a few settings and trying it out. Nothing radical.

**Comment by z1nc0r3** [comment]
Have you tried disabling the **Explorer** module?

**Comment by raindropsfromsky** [comment]
> Have you tried disabling the **Explorer** module?

If I turn off the **Explorer** plugin, FL stops showing results altogether!

**Comment by VictoriousRaptor** [comment]
Looks like something wrong with your clipboard. @taooceros Is it similar to the clipboard shortcut issue you've fixed before?

**Comment by raindropsfromsky** [comment]
> @raindropsfromsky Sorry, I'm confused about your issue. Can you please describe it a little bit? Can you start FL or not (crashing before starting)?

This happened only once, while I was tinkering with the plugins and settings.
FL simply disappeared and popped up this error.
I simply copied it and pasted it here.

It is not repeatable.

**Comment by raindropsfromsky** [comment]
> Could you check typing "reload plugin data" and see whether the error is reproducible?

I tried this, and got a confirmation message. FL works fine after that.

That said, I must also mention that after I got that error message a few days ago, I have been using FL heavily, but never faced any crash.

In other words, the crash seemed to be one-off.

**Comment by taooceros** [comment]
Yeah it seems like the toast notification is not available in your computer, but flow didn't use that much so it's not gonna affect your daily usage. Could you share which windows version you are using?

**Comment by raindropsfromsky** [comment]
I am using Windows 11 Home 64 bit

**Comment by jjw24** [comment]
I sometimes run into this error during startup in debug mode as well. @taooceros @VictoriousRaptor Have you also run into this error before?

**Comment by C0nfusion1** [comment]
I experience this issue. I uninstalled and reinstalled Flow Launcher. It crashes as soon as I start the setup.

**Comment by jjw24** [comment]
Taking a look at this issue, it appears to be from the [Bump ModernWpfUI from 0.9.4 to 0.9.6](https://github.com/Flow-Launcher/Flow.Launcher/commit/4a74ad0572ceaeb152c255ea12c29abac4dadc0d) change.

This causes flow's notification to error out, and in turn causes the following critical issues when error happens on user machine:
- Cannot install plugin because the install process uses notification, this also leaves behind just the zip file in the plugin folder
- When starting up flow, no notification of any plugin that failed to load
- Generally any other process that uses the notification will not be able to complete

Code location:

![image](https://user-images.githubusercontent.com/26427004/231013193-f660f0d2-d104-48c0-83f6-dccb478bda85.png)

For me only reproducible via debug mode while running Visual Studio. Error does not appear if running a build from the repo's ci pipeline.

Reproduce in Visual Studio debug mode:
Trigger notification, this can be by pressing F5 on the search window or install a plugin.

Downgrading ModernWpfUI from 0.9.6 to 0.9.4 and rebuild, trigger notification, the error no longer is thrown.

**Comment by jjw24** [comment]
Additional investigation to above:

I can replicate the issue from v1.12.0 onwards.

Removing the Programs plugin will also stop the error from happening, which leads me to think it could be related to this https://github.com/microsoft/CsWinRT/issues/623#issuecomment-742017274

**Comment by jjw24** [comment]
Traced the error to this change made to the way we load plugin dlls  https://github.com/Flow-Launcher/Flow.Launcher/pull/367, reverting to the commit before this change will also resolve the error.
